docs/quickstart-manually.md

```diff
@@ -1,66 +0,0 @@
-# Quickstart
-
-You can locally build and deploy `gitbase-playground` and its dependencies using [`docker-compose`](https://docs.docker.com/compose/install/)
-
-If you prefer to run `gitbase-playground` with [`docker-compose`](https://docs.docker.com/compose) (without taking care of the app dependencies), you can follow [the playground compose quickstart](quickstart.md)
-
-
-## Run bblfsh and gitbase Dependencies
```

Comment:
I think this section was really nice, can we keep it and link to if from other sections like `Integration Tests`, etc.

Also, probably a nitpick, but bash commands with multiple arguments looks much more readable with breaking up the lines with `\` and I suggest keeping like that.

Reply:
> I think this section was really nice, can we keep it and link to if from other sections like Integration Tests, etc.

I agree it can be very convenient to have in our documentation.

I think this fits better in the Contributing guide, where you want to run the project from sources. This way the main README has Docker Compose as the main suggested method to run it, and links to the other projects documentation if you still want to run them separately.

Change made in 1827f43.